Modules/Microsoft365DSC/Examples/Resources/AADAgreement/2-Update.ps1

```diff
@@ -0,0 +1,30 @@
+<#
+This example creates a Terms of Use Agreement that requires re-acceptance every 30 days on each device.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter(Mandatory = $true)]
+        [PSCredential]
+        $Credential
```

Comment:
Instead of defining $Credentials, we should be defining parameters $ApplicationId, $TenantId and $CertificateThumbprint. Then in the actual example below we should also be using those parameters instead of the $credential one.